Backend/utils/MentiiAuth.py

```diff
@@ -0,0 +1,83 @@
+from flask import g
+from itsdangerous import (TimedJSONWebSignatureSerializer as Serializer, BadSignature, SignatureExpired)
+from boto3.dynamodb.conditions import Key, Attr
+import hashlib
+import ConfigParser as cp
+
+class MentiiAuthentication:
+
+  def __init__(self):
+    return
+
+  @staticmethod
+  def build_user_object(userData):
+    '''
+    Builds a standard user object to return to the client for all authenticated calls
+    '''
+    #TODO: Eventually add role to this packet
+    return {'email': userData['email']}
```

Comment:
Ill add the check but if it has no email key by this stage then we have WAYYYY bigger issues to deal with lol